src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -57,75 +65,116 @@ def add_model_args(parser):
     )
 
 
-def add_profile_args(parser):
+def add_profile_args(parser, exclusive_group):
     parser.add_argument(
         "-b",
-        "--batch-size",
         type=int,
         default=1,
         required=False,
-        help="The batch size / concurrency to benchmark. (Default: 1)",
+        help="The batch size to benchmark. The default value is 1.",
     )
-    parser.add_argument(
-        "--input-length",
+    exclusive_group.add_argument(
+        "--concurrency",
         type=int,
-        default=128,
         required=False,
-        help="The input length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the concurrency value to benchmark.",
     )
     parser.add_argument(
-        "--output-length",
+        "--max-threads",
         type=int,
-        default=128,
+        default=16,
         required=False,
-        help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the maximum number of threads that will be "
+        "created for providing desired concurrency or request rate. "
+        "However, when running in synchronous mode,this value will be ignored. "
+        "The default value is 16.",
     )
-
-
-def add_endpoint_args(parser):
+    # TODO: necessary?
+    # parser.add_argument(
+    #     "--output-length",
+    #     type=int,
+    #     default=128,
+    #     required=False,
+    #     help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
+    # )
     parser.add_argument(
-        "--url",
-        type=str,
-        default="localhost:8001",
+        "--profile-export-file",
+        type=Path,
+        default="profile_export.json",
+        help="Specifies the path where the profile export will be "
+        "generated. By default, the profile export will not be "
+        "generated.",
+    )
+    exclusive_group.add_argument(
+        "--request-rate",
+        type=float,
         required=False,
-        help="URL of the endpoint to target for benchmarking.",
+        help="Sets the request rate for the load generated by PA. ",
     )
     parser.add_argument(
-        "--provider",
+        "--service-kind",
         type=str,
         choices=["triton", "openai"],
```

Comment:
Do we need to distinguish between triton infer and triton generate? @tgerdesnv

Reply:
Yes I think we will. TBD what the new generate 'kind' name will be. I think this existing mapping is fine, and I will add the new name if/when it is added to PA